tests/Plugins/utilities.py

```diff
@@ -71,7 +71,7 @@ def GetGithubUrl(remote_name: str = "origin") -> str:
     return repo_url.split(".git")[0]
 
 
-def CheckPATFileExists(github_pat_filename: Path):
+def CheckPATFileExists(github_pat_filename: Path, github_url: str = "https://github.com"):
```

Comment:
Is there code that invokes this method with a custom value for `github_url`? I did a quick search in this PR but didn't see it.

Reply:
It is being used in `GitHubEnterprises_EndToEndTest.py`. I added a comment which I tagged you in, the search must have failed because GitHub doesn't render files with large changes.